data/static/base_info.csv

```diff
@@ -61,7 +62,9 @@ United Nations World Food Programme (WFP),2013-06-11,Multilateral,wfp,70
 United States,2013-01-02,Government,unitedstates,27.25
 US Agency for International Development (USAID),2017-04-27,Government,usaid,33
 "World Bank, The",2011-03-14,Multilateral,worldbank,51.35
+World Health Organisation,2017-06-08,Multilateral,who,0
```

Comment:
Waiting for  confirmation on when WHO formally joined the Grand Bargain as a signatory

Reply:
Confirmation from WHO that they joined  1 Jan 2018 . Please recalculate baseline as of 1 January 2018.